realm-transformer/src/main/kotlin/io/realm/transformer/RealmTransformer.kt

```diff
@@ -147,16 +147,16 @@ class RealmTransformer(val project: Project) : Transform() {
             }
         }
 
-        val packages: Collection<String> = outputModelClasses.map {
+        val packages: Set<String> = outputModelClasses.map {
             it.packageName
-        }
+        }.toSet()
 
         val targetSdk: String? = GroovyUtil.getTargetSdk(project)
         val minSdk: String?  = GroovyUtil.getMinSdk(project)
 
-        if (disableAnalytics) {
+        if (!disableAnalytics) {
```

Comment:
Ouch, did that happen before or after https://realm.io/blog/realm-report-q4-2017-rise-of-kotlin/ i wonder

Reply:
Way after... this happened when the incremental transformer was added, which was in 5.2.0 (June this year).